src/libraries/sendtohelixhelp.proj

```diff
@@ -106,7 +106,8 @@
     <HelixPreCommand Include="set XHARNESS_LOG_WITH_TIMESTAMPS=true" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetOS)' == 'Browser'">
-    <HelixPreCommand Include="dotnet dev-certs https" />
+    <!-- this is alternative to `dotnet dev-certs https` but we don't have SDK installed on helix -->
```

Comment:
Can you maybe be a bit more elaborate in the doc and say that the cert is used by the local HTTPS Kestrel server which is spawned by XHarness?